apps/satsale/docker-compose.yml

```
@@ -0,0 +1,25 @@
+version: "3.7"
+
+services:
+  satsale:
+    image: satsale/satsale:0.2.9
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
+      - ${LND_DATA_DIR}:/lnd:ro
+    ports:
+      - 5000:5000
+    environment:
+      DATA_DIR: ${APP_DATA_DIR}
```

This should be the data dir inside the container not on the host.
```suggestion
      DATA_DIR: "/data"
```